sed/config/default.yaml

```diff
@@ -2,6 +2,14 @@ core:
   # The loader to use. The generic loader allows for loading various files types directly using dask.
   loader: generic
 
+  # If loader is set to flash, the following parameters are necessary
+  # Needs to be user defined
+  beamtime_id: None
+  year: None
+  # Default beamline is pg2 and instrument is hextof
+  beamline: pg2
+  instrument: hextof
```

Comment:
Maybe we rename these to flash_beamline and flash_instrument? Then it is clear they belong to the FLASH loader...

Reply:
These are no longer necessary. The initial insight to add them was to seperate the hextof and wespe instrument (and for future), even though they follow the same structure. The only difference being that wespe has two detectors. Since, sed can not support two dataframes simultaneously, for now, we agreed to just run two instances of sed for the two different detectors, and hence two different config files (I made appropriate changes to mark and save different detectors in the loader).

Reply:
See comment to flash_reader: Why do you not create one dataframe that has columns for both detectors?